usr/share/regolith-look/blackhole/typeface

```diff
@@ -0,0 +1,48 @@
+! Typeface: Source Code Pro
+#define typeface_bar_glyph  Source Code Pro Medium 9
+#define typeface_wm         pango:Source Code Pro Medium 9
+#define typeface_bar        pango:Source Code Pro Medium 9, FontAwesome 9
```

Comment:
There seems to be a font issue here.  I believe you're still using the 1.x fonts.  For 2.0 we've moved to fonts-nerd-fonts.  See https://github.com/regolith-linux/fonts-nerd-fonts/blob/debian/debian/control.  Essentially we no longer need to specify two sets of fonts for glphs and text.  They are both built into one.  Notice that the 2.0 font specification only specifies one font.

Reply:
(this applies to all font references in the look)